userdetails-plugin/grails-app/i18n/messages.properties

```diff
@@ -381,4 +381,7 @@ application.a11=Yes, you can access ALA restricted APIs using Python and R by im
 application.q12=Where can I learn more?
 application.a12=We have detailed documentation, including a listing of all available endpoints, on our <a href={0} target="_blank">API Docs Portal</a>.
 application.q13=I'm encountering issues. How can I contact the ALA?
-application.a13=For any questions or technical assistance, please contact our dedicated support team at <a href="mailto:support@ala.org.au">support@ala.org.au</a>.
\ No newline at end of file
+application.a13=For any questions or technical assistance, please contact our dedicated support team at <a href="mailto:support@ala.org.au">support@ala.org.au</a>.
+update.email=Update your email
+update.email.desc.1=Please enter your new email to update the email. The new email will receive a code to which needs to be submitted to verify the new email address.
```

Comment:
This should be moved into the userdetails-gorm/userdetails-cognito messages.properties with the same key, then remove the if branch in the createAccount.gsp below.